files/conf/k8s-addons

```diff
@@ -88,13 +88,15 @@ echo "kube-proxy successfully installed"
 # restart ds to apply config from configmap
 kubectl delete pods -l app.kubernetes.io/name=kube-proxy -n kube-system
 
+{{- if not .CalicoPolicyOnly }}
 # TODO: remove migrator once all clusters have been migrated to kubernetes datastore
 kubectl apply -f /srv/calico-crd-installer-rbac.yaml
 for job in calico-crd-installer calico-datastore-migrator-pre
```

Comment:
We need `calico-crd-installer` for policy-only and full Calico so we'll have to drop the `for` loop and split this in two parts.

Reply:
good point

Reply:
done